assignment/index.html

```diff
@@ -1,25 +1,41 @@
 <!DOCTYPE html>
 <html>
-<head>
+  <head>
     <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <!-- Change the <title> from "Page Title" to "Hello World" -->
     <title>Unit 1 Assignment</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
-</head>
-<body>
+  </head>
+  <body>
```

Comment:
Although it isn't necessary what you would expect, the typical convention is that elements which are direct children of the `<html>` element are _not_ indented.